mocca-client/src/test/java/com/paypal/mocca/client/MoccaClientQueryTest.java

```diff
@@ -48,6 +51,18 @@ public void queryRawTest() {
         assertEquals(result.getBar(), "far");
     }
 
+    @Test
+    public void queryInvalidRequest() {
+        try {
+            SampleResponseDTO result = client.getOneValidSample(new ValidatedRequestDTO(null, "zazzzzzzzzzzzzzzzzzzzzzzzzzz"));
```

Comment:
Let's add a `fail` after this line, to make sure an exception is thrown.

Reply:
We don't need to assign `SampleResponseDTO result` to the result of this method call.

Reply:
I'll fix it